client/my-sites/plans/jetpack-plans/product-lightbox/index.tsx

```diff
@@ -176,7 +208,7 @@ const ProductLightbox: React.FC< Props > = ( {
 				<div className="product-lightbox__sidebar">
 					<div className="product-lightbox__variants">
 						<div className="product-lightbox__variants-content">
-							{ shouldShowOptions && (
+							{ shouldShowOptions && ! isSocialProduct && (
```

Comment:
Doesn't this extra `! isSocialProduct` check render the changes to `shouldShowOptions` made earlier obsolete?

Reply:
I changed the variable name here and it looks like this now, let me know if that makes sense. 

```
{ shouldShowOptions && ! showVariantsWithoutRadioButtons && (
								<div className="product-lightbox__variants-options">
									<MultipleChoiceQuestion
										question={ PRODUCT_OPTIONS_HEADER[ product?.productSlug ] }
										answers={ variantOptions }
										selectedAnswerId={ product?.productSlug }
										onAnswerChange={ onChangeOption }
										shouldShuffleAnswers={ false }
									/>
								</div>
							) }
							{ shouldShowOptions && showVariantsWithoutRadioButtons ? ( .....
```